WowCharComparerWebApp/Controllers/User/LoginController.cs

```diff
@@ -8,5 +14,86 @@ public IActionResult Index()
         {
             return View();
         }
+
+        [Route("Login/PasswordRecovery")]
+        [HttpGet]
+        public IActionResult PasswordRecoveryView()
+        {
+            return View("PasswordRecovery");
+        }
+
+        [Route("Login/PasswordRecovery/Confirmation")]
+        public IActionResult PasswordResetConfirmationView(string userID)
+        {
+            ViewBag.Message = userID;
+            return View("PasswordChanged");
+        }
+
+        [HttpPost]
+        public IActionResult PasswordChanged(string newPassword, string newPasswordConfirmation, Guid userID)
+        {
+            List<string> passwordsToCheck = new List<string>();
+            {
+                passwordsToCheck.Add(newPassword);
+                passwordsToCheck.Add(newPasswordConfirmation);
+            }
+
+            List<(bool,string)> checkedPasswords = new List<(bool,string)>();
+            RegisterController registerController = new RegisterController();
```

Comment:
I suppose that is not a good idea to create additional instance of RegisterController. Try to replace that with RedirectToAction instead.